qiita_db/search.py

```diff
@@ -348,7 +348,7 @@ def _parse_study_search_string(self, searchstr,
             else:
                 header_info.append("sa.%s" % meta)
         # build the SQL query
-        sample_sql = ("SELECT r.sample_id,%s FROM qiita.required_sample_info "
+        sample_sql = ("SELECT r.sample_id,%s FROM qiita.study_sample "
                       "r JOIN qiita.sample_{0} sa ON sa.sample_id = "
```

Comment:
I know this isn't your code, but could you update this to use the `USING` convention? Thanks.